xdg-ninja.sh

```diff
@@ -42,7 +42,10 @@ init_constants() {
 
     BG_MAGENTA="\033[45m"
 }
-init_constants
+
+if test -t 0 && test "$TERM" != "dumb" && test "$TERM" != "xterm-mono" && test -n "$NO_COLOR" & test "$LS_COLORS" -gt 0; then
```

Comment:
I _think_ this should be `z` instead of `n`?
```suggestion
if test -t 0 && test "$TERM" != "dumb" && test "$TERM" != "xterm-mono" && test -z "$NO_COLOR" & test "$LS_COLORS" -gt 0; then
```
I don't have `$NO_COLOR` set, `test -z "$NO_COLOR"` gives me true, while `test -n "$NO_COLOR"` gives me false.